packages/worker/src/workflow/vm.ts

```diff
@@ -156,10 +85,11 @@ export class VMWorkflowCreator implements WorkflowCreator {
       throw new IllegalStateError('Isolate context provider was destroyed');
     }
     let context;
+    const globals = { AsyncLocalStorage, assert, __webpack_module_cache__: {} };
     if (this.hasSeparateMicrotaskQueue) {
```

Comment:
nit: At this point, we can actually remove this condition, since is <14.18 is no longer supported anyway.